src/hooks/use-import-export.tsx

```diff
@@ -84,6 +75,11 @@ export const ImportExportProvider = ( { children }: { children: React.ReactNode
 				return;
 			}
 
+			const INITIAL_IMPORT_STATE = {
```

Comment:
I considered adding those constants as dependencies, but it would complicate the code as they would need to be memorized, and they would need to use the current locale as a dependency. Is there any better way?

Reply:
yeah, I don't like we re-declare the same status, such as `const INITIAL_EXPORT_STATE` two times. 

The react way would be creating a hook that returns all those states that can be reused, those should be callbacks or memos so you don't need to add them to the dependencies array.

Another way, less reactish, but I think I would prefer is moving the constants outside of the function 

```
const INITIAL_EXPORT_STATE = {
	statusMessage: __( 'Starting export...' ),
	progress: 5,
};
const INITIAL_IMPORT_STATE = {
	statusMessage: __( 'Extracting backup…' ),
	progress: 5,
};

export const ImportExportProvider = ( { children }: { children: React.ReactNode } ) => {
....
```

We could also add `as const` at the end, so we make sure the object is not modified later.

```
const INITIAL_EXPORT_STATE = {
	statusMessage: __( 'Starting export...' ),
	progress: 5,
} as const;
```

Reply:
Yes, translations wouldn't work if we defined those constants in the global scope.

As we chatted,I simplified this code and removed constants. One of statuses didn't even need the constant as it was used once. The other one is now used two times which is acceptable.

We can always refactor and implement what you proposed:

> The react way would be creating a hook that returns all those states that can be reused, those should be callbacks or memos so you don't need to add them to the dependencies array.